scripts/extractCodeSnippets.js

```diff
@@ -24,7 +24,14 @@ function findAllCodeSnippetsInJsDoc(jsDoc) {
       if (!codeSnippets.has(operationId)) {
         codeSnippets.set(operationId, []);
       }
-      codeSnippets.get(operationId).push(stripMarkdownCodeSnippet(rawCode));
+
+      try {
+        codeSnippets.get(operationId).push(stripMarkdownCodeSnippet(rawCode));
```

Comment:
What failed?

Reply:
When the comment above the method is not in a format that this tool is expecting, it is failing without telling why or showing a useful error message. It took me quite some time to figure out why (tnx to @polomani )